src/server-extension/eigenlayer.ts

```diff
@@ -32,177 +32,27 @@ export class EigenLayerResolver {
-                '0.lastUpdateBlockTime': string[]
-                '2.shares': string[]
-              }
-            }
-            json: {
-              shares: string
-              lastUpdateBlockNumber: string
-              integratedShares: string
-              lastUpdateBlockTime: string
-              strategyAddress: string
-              totalIntegratedShares: string
-              totalShares: string
-            }[]
-          }
-        }
-      },
-      {
-        result: {
-          data: {
-            json: {
-              tokenPrices: {
-                'origin-ether': { usd: number }
-                'wrapped-beacon-eth': { usd: number }
-                'coinbase-wrapped-staked-eth': { usd: number }
-                'mantle-staked-ether': { usd: number }
-                'staked-frax-ether': { usd: number }
-                ankreth: { usd: number }
-                ethereum: { usd: number }
-                'liquid-staked-ethereum': { usd: number }
-                'staked-ether': { usd: number }
-                'stader-ethx': { usd: number }
-                'stakewise-v3-oseth': { usd: number }
-                sweth: { usd: number }
-                'rocket-pool-eth': { usd: number }
-              }
-            }
-          }
-        }
-      },
-      {
-        result: {
-          data: {
-            json: {
-              globalStats: {
-                balance: number
-                validators: number
-                points: number
-              }
-              podInfo: { address: null; slashingPresent: null }
-              podStats: { balance: number; validators: number; points: number }
-            }
-          }
-        }
-      },
-    ]
-  >(API, { withCredentials: false })
-
   const sData = response.data?.[0]['result']['data']['json']
-  const pData = response.data?.[1]['result']['data']['json']['tokenPrices']
-  const gData = response.data?.[2]['result']['data']['json']['globalStats']
+  const gData = response.data?.[1]['result']['data']['json']['globalStats']
 
   let uPoints = 0
   let gPoints = 0
   const nPoints = gData['points']
-  let block = '0'
-  let lastTimestamp = '0'
 
   // Process staking data
   for (const stake of sData) {
     const uIshares = parseFloat(stake['integratedShares'])
     const gIshares = parseFloat(stake['totalIntegratedShares'])
     uPoints += uIshares / 1e18 / 3600
     gPoints += gIshares / 1e18 / 3600
-    block = stake['lastUpdateBlockNumber']
-    lastTimestamp = stake['lastUpdateBlockTime']
   }
 
-  return {
-    a: parseInt(block),
-    b: parseInt(lastTimestamp),
-    c: ADDRESS,
-    d: parseFloat(uPoints.toFixed(2)),
-    e: parseFloat(gPoints.toFixed(2)),
-    f: parseFloat(nPoints.toFixed(2)),
-    g: parseFloat((uPoints + gPoints + nPoints).toFixed(2)),
-    h: JSON.stringify(sData),
-    i: JSON.stringify(pData),
-    j: JSON.stringify(gData),
-    k: lastTimestamp,
-  }
+  return parseFloat(uPoints + gPoints + nPoints)
```

Comment:
Are we just using this to calculate the total EL points issued? If so, we can exclude the integratedShares calculation above (and uIshares), which _should_ be 0 for the burn address. totalIntegratesShares will give us the number of points issued globally for a specific asset.

Reply:
Why is it these numbers don't really match up with some Dune dashboards I found?

Reply:
So these are the computed values from a call done today:
- **integratedShares** total: `0`
- **totalIntegratedShares** total: `1889418940.5178962`
- **totalEigenLayerPoints**: `2760903266.9178963`

I'm unsure about the accuracy of those figures, but I'll remove the `integratedShares` as Jon suggests since it's 0

Reply:
and this is the result of the api call (made at a slightly diiferent time, values may differ a bit):
```
  {
    "result": {
      "data": {
        "json": [
          {
            "integratedShares": "0",
            "totalIntegratedShares": "591325531775699791706076962426",
            "strategyAddress": "0x1bee69b7dfffa4e2d53c2a2df135c388ad25dcd2"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "3618125569392120758887339885806",
            "strategyAddress": "0x93c4b944d05dfe6df7645a86cd2206016c51564d"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "430951472695120618283813492928",
            "strategyAddress": "0x54945180db7943c0ed0fee7edab2bd24620256bc"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "103479846355946643292772859322",
            "strategyAddress": "0x57ba429517c3473b6d34ca9acd56c0e735b94c02"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "844027719751683724843456703162",
            "strategyAddress": "0x0fe4f44bee93503346a3ac9ee5a26b130a5796d6"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "85720397487764154451750550744",
            "strategyAddress": "0xa4c637e0f704745d182e4d38cab7e7485321d059"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "216269012744599270027156135627",
            "strategyAddress": "0x7ca911e83dabf90c90dd3de5411a10f1a6112184"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "6641187748329563847681085977",
            "strategyAddress": "0x13760f50a9d7377e4f20cb8cf9e4c26586c658ff"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "395237507472431210821344149128",
            "strategyAddress": "0x9d7ed45ee2e8fc5482fa2428f15c971e6369011d"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "52437846951314171943566150564",
            "strategyAddress": "0x8ca7a5d6f3acd3a7a8bc468a8cd0fb14b6bd28b6"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "19913060806473388950148804179",
            "strategyAddress": "0xae60d8180437b5c34bb956822ac2710972584473"
          },
          {
            "integratedShares": "0",
            "totalIntegratedShares": "371194630730682468940164751796",
            "strategyAddress": "0x298afb19a105d59e74658c4c334ff360bade6dd2"
          }
        ]
      }
    }
  },
  {
    "result": {
      "data": {
        "json": {
          "podInfo": { "address": null, "slashingPresent": null },
          "podStats": { "balance": 0, "points": 0, "validators": 0 },
          "globalStats": {
            "balance": 1137376,
            "points": 859004213.76,
            "validators": 35543000000000
          }
        }
      }
    }
  },
```

Reply:
For now, these numbers are pretty good, even if they don't match the two Dune dashboards estimating the points. They're directionally accurate historically and line up with impersonation.

The cleanup looks great!